modules/manual.nix

```
@@ -83,13 +83,25 @@ in
         Thanks!
       '';
     };
+    
+    manual.optionList.enable = mkOption {
```

Instead of `optionList` you could simply call this `json`:
```suggestion
    manual.json.enable = mkOption {
```